source/includes/_embed2.md.erb

```diff
@@ -442,6 +455,27 @@ Removes Embed2 from your page.
 
 Can be used to programatically open / close the Chat window. This method cannot be used with the `parentElement` option.
 
+#### triggerCampaign
+`triggerCampaign(campaignKey: string, options: object): Promise<void>;`
```

Comment:
It does return a promise, but it's not necessary or used anywhere.

Reply:
I'm just following the pattern in the rest of the docs for consistency

Reply:
yeah, but there's a point to the other promises. I think people explicitly _shouldn't_ wait for this one :)

Reply:
I'm a big fan of consistency, but helpfulness comes first!

Reply:
Well I mean, any other return type would be incorrect - I guess you're suggesting just leaving the return type off for this one?

Reply:
Maybe it's not useful to wait for the promise, but I don't see why we need to discourage it

Reply:
If they use `await` then this will introduce unnecessary latency into their application. I would just add that this is a `fire & forget` operation that does not return any data.

Reply:
Isn't that already indicated by the `void` though?

Reply:
Hm yeah, you're right!